src/locales/cy/translation.json

```diff
@@ -505,6 +509,12 @@
         "description": "Rheoli eich prentisiaid, darparwyr hyfforddiant, recriwtio a chyllid.",
         "link_text": "Ewch i'ch cyfrif gwasanaeth prentisiaeth",
         "link_href": "https://accounts.manage-apprenticeships.service.gov.uk/service"
+      },
+      "tya4DoMpw_B7FK5YvuMAj3asc0A": {
```

Comment:
is there an integration version of the client?

Reply:
We've not been sent an integration client ID for this service. I know there's been some testing in production before they went live, so that's probably why. I'll ask around and add integration details in a later PR if there is one